types/passport-jwt/index.d.ts

```diff
@@ -48,7 +48,7 @@ export interface JwtFromRequestFunction {
 }
 
 export interface SecretOrKeyProvider {
-    (request: Request, rawJwtToken: any, done: (err: any, secretOrKey: string | Buffer) => void): void;
+    (request: Request, rawJwtToken: any, done: (err: any, secretOrKey?: string | Buffer) => void): void;
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ddb440a13bc5d4c5a430cbbc79546c2f376abd2d/types/express-jwt/index.d.ts#L28-L30

`secretOrKey` is optional paramaters.